application-templates/starter/package.json

```diff
@@ -19,17 +19,13 @@
     "@commercetools-frontend/i18n": "16.0.0",
     "@commercetools-frontend/mc-http-server": "16.0.0",
     "@commercetools-frontend/permissions": "16.0.0",
-    "@commercetools-uikit/flat-button": "10.13.0",
-    "@commercetools-uikit/icons": "10.13.0",
-    "@commercetools-uikit/spacings": "10.13.0",
-    "@commercetools-uikit/text": "10.13.0",
+    "@commercetools-frontend/ui-kit": "10.13.0",
     "prop-types": "15.7.2",
     "react": "16.12.0",
     "react-apollo": "3.1.3",
     "react-dom": "16.12.0",
     "react-intl": "3.4.0",
     "react-redux": "7.1.3",
-    "react-router": "5.1.2",
```

Comment:
It's included as a dep of `react-router-dom`

<img width="269" alt="image" src="https://user-images.githubusercontent.com/1110551/73758020-62540380-476a-11ea-97d4-25f627036c4d.png">